docs/book_files/redirect.properties

```diff
@@ -0,0 +1 @@
+blockstg-csi-driver/* = stg-block-csi-driver/*
```

Comment:
it seems GitHub expects a new line at the end of file here.
could we add it? or will it break something?